source/agent/addons/quic/QuicTransportConnection.cc

```diff
@@ -137,10 +161,42 @@ NAN_METHOD(QuicTransportConnection::close)
         }
         auto maybeReason = Nan::Get(Nan::To<v8::Object>(info[0]).ToLocalChecked(), Nan::New<v8::String>("reason").ToLocalChecked());
         if (!maybeReason.IsEmpty()) {
-            auto reasonObj =maybeReason.ToLocalChecked();
+            auto reasonObj = maybeReason.ToLocalChecked();
             Nan::Utf8String reasonStr(Nan::To<v8::String>(reasonObj).ToLocalChecked());
             reason = std::string(*reasonStr);
         }
     }
     obj->m_session->Close(closeCode, reason.c_str());
-}
\ No newline at end of file
+}
+
+void QuicTransportConnection::onFrame(const owt_base::Frame& frame)
+{
+    if (frame.format != owt_base::FRAME_FORMAT_RTP) {
+        ELOG_WARN("WebTransport datagram output only supports RTP packets. Received frame type %d. Discard this frame.", frame.format);
+        return;
+    }
+    if (!m_session) {
+        ELOG_WARN("WebTransport session is nullptr.");
```

Comment:
It's unexpected, but better not put `m_session->SendOrQueueDatagram(frame.payload, frame.length);` after `!m_session`

Reply:
Added return if `m_session` doesn't exist.